docs/navigation.md

```diff
@@ -0,0 +1,114 @@
+
+
+## Schemas
+
+The Schemas displays the schemas stored in the `components.schemas` section. This does not mean that every schema is used in the document. You can have it defined but it may not be used by references in the `message.payload` field.
+
+
+# Editor
+
+The Editor section is where you edit the AsyncAPI file. As you edit, you will see the changes reflect dynamically in the HTML preview and Block Visualizer section.
+
+Click on the individual elements of the Navigation button to jump to those sections in HTML preview and Block Visualizer section.
+
+## Diagnostics Window
+
+The Editor also has resizeable Diagnostics window. It shows a quick count of number of 
+- errors (in red),
+- warnings (in yellow),
+- informations (in blue),
+- hints (in green)
+by its title. The description of the diagnostics with line number is displayed in a table.
+
+To the right side of the title bar there are three indicators of the Diagnostics window: 
+
+- Valid/Invalid - It is green if the syntax is green and red when invalid.
+- Autosave - It shows whether Autosave is on or off.
+- Type - Shows the language in which the API is written: `JSON` or `YAML`.
+
+Below the title bar there is a control panel. You can the buttons to:
+
+- Hide and show erros
+- Hide and show warnings
+- Hide and show information messages
+- Hide and show hints
+
+There is also search bar to search that aids in finding issues by their names. You will see Settings icon to the right. It will take you to the governance of Diagnostic's settings window.
+
+# HTML Preview
+
+This section displays the preview of everything you have written in the code editor. It uses different fonts, colors, and styles for readability. Each section has a different card. 
+
+It starts with displaying the title, version, description of the API. It renders the formatting too. You can expand each element too see the `key: value` pair of each element, type, payload, and message.
+
+
+# Blocks visualizer
+
+Blocks visualizer gives you a quick overview of your API. Unlike HTML preview, Blocks visualizer gives a diagrammatic visual of your API. 
+
+Title card is at the top-left corner of the Block visualizer window. Followed by the Operations card.  You can visualize channels, published messages, payload, type,  patterns, and relationship between the components.
+
+The Blocks visualizer has two set of controls:
+
+- In the top right corner of the window you will find the play and reset button. Play to view live relationship between the components and its flow. Reset to get back to the default view.
+
+- To bottom left of the window there are four controls: zoom in, zoom out, full-screen, toogle interactivity. You zoom the cards in and out or view them in full-screen mode. To prevent the cards from shifting use the toggle interactivity control.
```

Comment:
```suggestion
- To the bottom left of the window there are four controls: zoom in, zoom out, full-screen, and toogle interactivity. You zoom the cards in and out or view them in full-screen mode. To prevent the cards from shifting use the toggle interactivity control.
```